src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -172,34 +168,4 @@ private static DetailNode parseFile(File file) throws IOException {
             System.getProperty("file.encoding", "UTF-8"));
         return parseJavadocAsDetailNode(text.getFullText().toString());
     }
-
-    /**
-     * Creates DetailAST block comment to pass it to the Javadoc parser.
-     * @param content comment content.
-     * @return DetailAST block comment
-     */
-    private static DetailAST createFakeBlockComment(String content) {
```

Comment:
> ok, I think percentage should be decreased in config

yes, unfortunate calculation, ok to decrease, but should be done in separate PR with change only for classes that cause decrease.